### PR TITLE
feat(orb-ui): re-enable profile page

### PR DESCRIPTION
**Pull request by gpazuch**
Reenable Profile Page

### CODE REVIEW

**Comment by gpazuch** [comment]
@iatdaitan @weyrick -  should we leave the other fields from Mainflux as well, or allow only for name, email and company only?

image

**Comment by weyrick** [comment]
> @iatdaitan @weyrick - should we leave the other fields from Mainflux as well, or allow only for name, email and company only?

Just name, email, and company, yes

**Comment by gpazuch** [comment]
Fields are now correct.
@iatdaitan suggestions on layout and sizes for each field are appreciated.
I'm thinking of centering it, downsize to col-10, one field per line, col-12 for each field.

**Comment by iatdaitan** [comment]
> Fields are now correct. @iatdaitan suggestions on layout and sizes for each field are appreciated. I'm thinking of centering it, downsize to col-10, one field per line, col-12 for each field.

I would say having two columns col-6 one to hold "User Information" and another one to hold "Change Password". What do you think about it?

**Comment by gpazuch** [comment]
image
@iatdaitan how do you like it?

I still need to check why email and fullname are not sent in userServices.getProfile() and if there is such a route to retrieve such info.

**Comment by iatdaitan** [comment]
> image @iatdaitan how do you like it?
> 
> I still need to check why email and fullname are not sent in userServices.getProfile() and if there is such a route to retrieve such info.

@gpazuch  I really do like that way. Do you need any help from @dscabral or @mclcavalcante  regarding the service?

**Comment by gpazuch** [comment]
image
image
@dscabral || @mclcavalcante could you help me out on this?
Retrieving the user to edit on profile page returns these fields, and I wonder if we have another route to query user for Ord

**Comment by dscabral** [comment]
image
These are the users endpoint... 
On profile the unique mapped fields are Id and Email... any other field should be sended as metadata

**Comment by gpazuch** [comment]
@dscabral my question is regarding how to retrieve user name. I must request it and pass the fields I need listed as metadata of GET request?
image
I noticed I wasn't sending the fullName property on this POST.

image

**Review by iatdaitan — DISMISSED** [review]
Good work Guilhermo

File: ui/src/app/pages/pages-routing.module.ts [hunk 1]
```diff
@@ -23,13 +23,19 @@ import { AgentGroupAddComponent } from 'app/pages/fleet/groups/add/agent.group.a
...
 
 const children = [
   {
     path: 'home',
     data: {breadcrumb: 'Home'},
     component: DashboardComponent,
   },
+  {
+    path: 'profile',
+    data: {breadcrumb: 'Home'},
```
[the remark this answers is not in the document]
Comment (gpazuch) [reply ui/src/app/pages/pages-routing.module.ts]: Done

**Review by manrodrigues — APPROVED** [review]

**Pull request merged**